system-test/fixtures/google-gax-packaging-test-app/protos/protos.json

```diff
@@ -0,0 +1,1281 @@
+{
```

Comment:
Please remove `google-gax.tgz` from this change (the file above) - we don't need to commit it.